### PR TITLE
zh/en: add br notes to remind user of KMS TiKV encryption at rest

### LINKED ISSUE #2220: br: volume snapshot backup and restore may never success when KMS is disabled
## Error Report

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the URL/path of the document related to this issue?
according to https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/backup-to-aws-s3-by-snapshot.

When backup cluster with TiKV encrypted by AWS KMS, restore may failure when KMS is disabled at point of time restore

2. How would you like to improve it?

Provided a reminder of customer never disable kms when backup is existed.

### PR DESCRIPTION
**Pull request by fengou1**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
a new note to remind user of KMS TiKV encryption at rest
close #2220 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/backup-to-aws-s3-by-snapshot.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ summary: 介绍如何基于 EBS 卷快照使用 TiDB Operator 备份 TiDB 集群
 
 > **注意：**
 >
-> 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+> - 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+>
+> - 对于集群已经打开 TiKV KMS [静态加密](https://docs.pingcap.com/zh/tidb/dev/encryption-at-rest#tikv-静态加密)，需要在恢复阶段确保 AWS KMS 服务中主密钥或者旧的主密钥是可用的状态。
```
Comment (shichun-0415) [inline zh/backup-to-aws-s3-by-snapshot.md:33]:
```suggestion
> - 如果集群打开了 TiKV KMS [静态加密](https://docs.pingcap.com/zh/tidb/dev/encryption-at-rest#tikv-静态加密)，则需要在恢复阶段确保 AWS KMS 服务中主密钥或者旧的主密钥是可用的状态。
```

Comment (shichun-0415) [reply zh/backup-to-aws-s3-by-snapshot.md:33]: `主密钥或者旧的主密钥` 这个表达有点怪，逻辑上，`旧的主密钥`包含在`主密钥` 范围内。

Comment (fengou1) [reply zh/backup-to-aws-s3-by-snapshot.md:33]: 把旧的主密钥拿掉？

Comment (fengou1) [reply zh/backup-to-aws-s3-by-snapshot.md]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2225/commits/4a2b220f073b166c5f912465b19388772d4eac64

File: zh/backup-to-aws-s3-by-snapshot.md [hunk 2]
```diff
@@ -28,7 +28,9 @@ summary: 介绍如何基于 EBS 卷快照使用 TiDB Operator 备份 TiDB 集群
 
 > **注意：**
 >
-> 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+> - 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+>
+> - 如果集群打开了 TiKV KMS [静态加密](https://docs.pingcap.com/zh/tidb/dev/encryption-at-rest#tikv-静态加密)，则需要在恢复阶段确保 AWS KMS 服务中主密钥是可用的状态。
```
Comment (WangLe1321) [inline zh/backup-to-aws-s3-by-snapshot.md:33]: 这里需不需要写上静态加密的配置需要跟备份时保持一致？还是我们后续自己给用户加上静态加密的配置？

Comment (fengou1) [reply zh/backup-to-aws-s3-by-snapshot.md:35]: fixed in https://github.com/pingcap/docs-tidb-operator/pull/2225/commits/26244a420f317ae1e118d26238b81079d2ec382b

**Review by WangLe1321 — APPROVED** [review]
LGTM

File: zh/backup-to-aws-s3-by-snapshot.md [hunk 3]
```diff
@@ -28,7 +28,11 @@ summary: 介绍如何基于 EBS 卷快照使用 TiDB Operator 备份 TiDB 集群
 
 > **注意：**
 >
-> 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+> - 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+>
+> - 推荐集群备份和恢复的 CRD 需要保持一致，此功能要求 TiKV 备份和恢复配置一致。可以从备份目标 S3 中，下载 backupmeta 文件，检查 kubernetes.crd_tidb_cluster.spec, 确保恢复时 TiKV 配置与备份时一致。
```
Comment (shichun-0415) [inline zh/backup-to-aws-s3-by-snapshot.md:33]:
```suggestion
> - 要进行卷快照备份，需要确保 TiKV 备份和恢复时的配置一致。可以从备份目标 S3 中，下载 `backupmeta` 文件，检查 `kubernetes.crd_tidb_cluster.spec`， 确保恢复时 TiKV 的配置与备份时一致。
```

File: en/backup-to-aws-s3-by-snapshot.md [hunk 4]
```diff
@@ -28,7 +28,9 @@ If you have any other requirements, select an appropriate backup method based on
 
 > **Note:**
 >
-> After you upgrade the TiDB cluster from an earlier version to v6.5.0, you might find the volume snapshot backup does not work. To address this issue, see [Backup failed after an upgrade of the TiDB cluster](backup-restore-faq.md#backup-failed-after-an-upgrade-of-the-tidb-cluster).
+> - After you upgrade the TiDB cluster from an earlier version to v6.5.0, you might find the volume snapshot backup does not work. To address this issue, see [Backup failed after an upgrade of the TiDB cluster](backup-restore-faq.md#backup-failed-after-an-upgrade-of-the-tidb-cluster).
+> - To perform volume snapshot restore, ensure that the TiKV configuration is consistent in backup and restore. To check consistency, download the `backupmeta` file from the backup file stored in Amazon S3, and check the filed `kubernetes.crd_tidb_cluster.spec`. If this field is inconsistent, you can modify the TiKV configuration based on [Configure a TiDB Cluster on Kubernetes](configure-a-tidb-cluster.md).
+> - If [Encryption at Rest](https://docs.pingcap.com/tidb/dev/encryption-at-rest) is enabled for TiKV KMS, ensure that the primary key is enabled during restore.
```
Comment (ran-huang) [inline en/backup-to-aws-s3-by-snapshot.md:33]:
```suggestion
> - If [Encryption at Rest](https://docs.pingcap.com/tidb/stable/encryption-at-rest) is enabled for TiKV KMS, ensure that the master key is enabled for AWS KMS during restore.
```
master key: https://docs.aws.amazon.com/streams/latest/dev/permissions-user-key-KMS.html

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: zh/backup-to-aws-s3-by-snapshot.md [hunk 5]
```diff
@@ -28,7 +28,9 @@ summary: 介绍如何基于 EBS 卷快照使用 TiDB Operator 备份 TiDB 集群
 
 > **注意：**
 >
-> 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+> - 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+> - 要进行卷快照恢复，需要确保恢复时 TiKV 的配置与备份时的配置一致。可以从备份目标 S3 中，下载 `backupmeta` 文件，检查 `kubernetes.crd_tidb_cluster.spec` 字段，确认 TiKV 的配置是否一致。如果不一致，可参考[在 Kubernetes 中配置 TiDB 集群](configure-a-tidb-cluster)修改 TiKV 的配置。
+> - 如果集群打开了 TiKV KMS [静态加密](https://docs.pingcap.com/zh/tidb/dev/encryption-at-rest#tikv-静态加密)，则需要在恢复阶段确保 AWS KMS 服务已启用主密钥。
```
Comment (shichun-0415) [inline zh/backup-to-aws-s3-by-snapshot.md:33]:
```suggestion
> - 如果集群打开了 TiKV KMS [静态加密](https://docs.pingcap.com/zh/tidb/stable/encryption-at-rest#tikv-静态加密)，则需要在恢复阶段确保 AWS KMS 服务已启用主密钥。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: zh/backup-to-aws-s3-by-snapshot.md [hunk 6]
```diff
@@ -28,7 +28,9 @@ summary: 介绍如何基于 EBS 卷快照使用 TiDB Operator 备份 TiDB 集群
 
 > **注意：**
 >
-> 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+> - 集群从低于 v6.5.0 版本升级到 v6.5.0 时，可能无法进行卷快照备份。详细解决办法见[升级后备份无法工作](backup-restore-faq.md#升级后备份无法工作)。
+> - 要进行卷快照恢复，需要确保恢复时 TiKV 的配置与备份时的配置一致。可以从备份目标 S3 中，下载 `backupmeta` 文件，检查 `kubernetes.crd_tidb_cluster.spec` 字段，确认 TiKV 的配置是否一致。如果不一致，可参考[在 Kubernetes 中配置 TiDB 集群](configure-a-tidb-cluster)修改 TiKV 的配置。
```
Comment (shichun-0415) [inline zh/backup-to-aws-s3-by-snapshot.md:32]:
```suggestion
> - 要进行卷快照恢复，需要确保恢复时 TiKV 的配置与备份时的配置一致。可以从备份目标 S3 中，下载 `backupmeta` 文件，检查 `kubernetes.crd_tidb_cluster.spec` 字段，确认 TiKV 的配置是否一致。如果不一致，可参考[在 Kubernetes 中配置 TiDB 集群](configure-a-tidb-cluster.md)修改 TiKV 的配置。
```

**Pull request merged**